docs/remote_inference_blueprints/cohere_connector_embedding_blueprint.md

```diff
@@ -24,7 +24,9 @@ POST /_plugins/_ml/connectors/_create
       "headers": {
         "Authorization": "Bearer ${credential.cohere_key}"
       },
-      "request_body": "{ \"texts\": ${parameters.texts}, \"truncate\": \"${parameters.truncate}\", \"model\": \"${parameters.model}\" }"
+      "request_body": "{ \"texts\": ${parameters.prompt}, \"truncate\": \"${parameters.truncate}\", \"model\": \"${parameters.model}\" }", 
+      "pre_process_function": "connector.pre_process.cohere.embedding",
+			"post_process_function": "connector.post_process.cohere.embedding"
```

Comment:
Can we fix the indentation here?

Reply:
@ylwu-amzn should we add these two fields in all the connector blueprints?

Reply:
Not all, these two functions should be added on demand. For text embedding model, they are usually used for neural search, so they need these two functions.